transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java

```diff
@@ -85,7 +85,10 @@ public void before() {
         page.navigate();
     }
 
-    private static String[] TRANSATIONS_XID = new String[]{
+    /**
+     * Unique identifiers of prepared transactions as they appear in UI
+     */
+    private static String[] TRANSACTIONS_XID = new String[]{
```

Comment:
Well, since the journal is just a bunch of binary files, I did not extract ids from it, but I used ids displayed in Web Console instead. For future updates, it needs to be taken from there again after the prepared journal is installed.